README.md

```diff
@@ -110,6 +110,34 @@ module.exports = [{
 }]
 ```
 
+#### `link` hooks
+If you're mastering your plugin which requires some user input (for example, google analytics token), you can use rnpm's prelink and postlink hooks. For instance:
+
+```json
+"rnpm": {
+  "hooks": {
+    "prelink": "./bin/input.js"
+  }
+}
+```
+
+Let's assume we have a `bin/input.js` executable:
+
+```javascript
+const inquirer = require('inquirer');
+
+inquirer.prompt([{
+  message: 'Please, insert here your GA token',
+  name: 'GAToken'
+}], (answers) => {
+  // Do something with answers.GAToken
```

Comment:
I don't agree with you. Hooks never pass it's results to main func body, they always works like this (see npm example). If you want to store settings - you can write them somewhere before/after plugin installation.